src/main/java/frc/robot/subsystems/GripperSubsystem.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import frc.robot.RobotConstants;
+
+public class GripperSubsystem extends EntechSubsystem{
+
+	private DoubleSolenoid gripperSolenoid;	
+	private int gripperSolenoidCounter;
+	private GripperState gripperState;
+	private boolean enabled = false;
+	private final int SOLENOID_HIT_COUNT = 10;
+	
+	public enum GripperState { kClose, kOpen }
+	
+	@Override
+	public void initialize() {
+		if (enabled ) {
+			gripperSolenoid = new DoubleSolenoid(PneumaticsModuleType.CTREPCM, RobotConstants.PNEUMATICS.GRIPPER_OPEN, RobotConstants.PNEUMATICS.GRIPPER_CLOSE);
+		}
+		
+	}
+	
+	@Override
+	public void periodic() {
+	  if ( enabled ) {
+	      if (gripperSolenoidCounter < SOLENOID_HIT_COUNT) {
+	          gripperSolenoidCounter += 1;
+	          if (gripperState == GripperState.kOpen) {
+	              gripperSolenoid.set(DoubleSolenoid.Value.kForward);
+	          } else if (gripperState == GripperState.kClose) {
+	              gripperSolenoid.set(DoubleSolenoid.Value.kReverse);
+	          } else {
+	              gripperSolenoid.set(DoubleSolenoid.Value.kOff);
+	          }
+	      }		  
+	  }
```

Comment:
This looks like my code; but I think there needs to be an else block on the SOLENOID_HIT_COUNT if statement that sets the gripperSolenoid.set(kOff);